examples/app/index.ts

```diff
@@ -105,7 +118,13 @@ toolSelectElements[0].addEventListener('input', (e: any) => {
   highlightTool(0);
 });
 
-toolSelectElements[1].addEventListener('click', (e: any) => {
+toolSelectElements[1].addEventListener('input', (e: any) => {
```

Comment:
How about setting the type of event object to `MouseEvent` rather than `any`?

Reply:
good point~!
i'll edit. 😄